app/livechat/server/lib/Livechat.js

```diff
@@ -430,7 +431,10 @@ export const Livechat = {
 	forwardOpenChats(userId) {
 		LivechatRooms.findOpenByAgent(userId).forEach((room) => {
 			const guest = LivechatVisitors.findOneById(room.v._id);
-			this.transfer(room, guest, { departmentId: guest.department });
+			const user = Users.findOneById(userId);
+			const { _id, username, name } = user;
+			const transferredBy = normalizeTransferredByData({ _id, username, name }, room);
+			this.transfer(room, guest, { roomId: room._id, transferredBy, department: LivechatDepartment.findOneById(guest.department) });
```

Comment:
You don't need to read the model here, you can do it inside the `Livechat.transfer`.